config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml

```diff
@@ -9,24 +9,13 @@ spec:
     value: profile_tasks
   - name: ANSIBLE_FORCE_COLOR
     value: "True"
-  networkAttachments:
-  - ctlplane
   nodeTemplate:
     ansible:
-      ansiblePort: 22
-      ansibleUser: cloud-admin
       ansibleVars:
-        ctlplane_dns_nameservers:
-        - 192.168.122.1
-        ctlplane_gateway_ip: 192.168.122.1
-        ctlplane_host_routes:
-        - ip_netmask: 0.0.0.0/0
-          next_hop: 192.168.122.1
-        ctlplane_mtu: 1500
-        ctlplane_subnet_cidr: 24
-        dns_search_domains: []
         timesync_ntp_servers:
         - hostname: pool.ntp.org
+        edpm_chrony_ntp_servers:
+        - pool.ntp.org
```

Comment:
this was removed in https://github.com/openstack-k8s-operators/edpm-ansible/commit/9d8a7b2497cd26de25a61dfa4c6965a5f4b0a007, so the var can be dropped from here.

Reply:
ack, done.